ui/dashboards/src/components/TimeRangeControls/TimeRangeControls.tsx

```diff
@@ -46,8 +50,8 @@ export function TimeRangeControls() {
 
   return (
     <>
-      <DateTimeRangePicker timeOptions={TIME_OPTIONS} value={timeRange} onChange={setTimeRange} />
-      <RefreshIconButton aria-label="Refresh Dashboard" onClick={refresh}>
+      <DateTimeRangePicker timeOptions={TIME_OPTIONS} value={timeRange} onChange={setTimeRange} height={height} />
+      <RefreshIconButton aria-label="Refresh Dashboard" onClick={refresh} sx={height ? { height } : {}}>
```

Comment:
optional nit: Do you need this ternary here? I _think_ if height is undefined, it won't set it, but I'm not sure if there's anything underlying this that would get clobbered by that `undefined`.

Reply:
ah yeah, seems like we can get away without it, so let's do that (testing it, it doesn't seem to add `height: undefined` to the CSS)